server/issue_handler.go

```diff
@@ -33,6 +33,7 @@ func (s *Server) issueEventHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
+	defer r.Body.Close()
```

Comment:
This seems unrelated?

Reply:
Not specifically related with the check but it's an error haven't been discovered yet :)

Reply:
I don't believe we need to close the body in server response handlers. This is only applicable for clients making the requests.

> // The Server will close the request body. The ServeHTTP
    // Handler does not need to.

Reply:
I see, didn't know that. Thanks 👍